config/routes.rb

```diff
@@ -1,8 +1,24 @@
 Rails.application.routes.draw do
-  devise_for :users 
+  devise_for :users, controllers: {
+    registrations: 'users/registrations',
+    sessions: 'users/sessions'
+  }
 
   resources :dashboard
   get '/hosting-dashboard', to: 'dashboard#dashboard'
   root 'static_pages#home'
   resources :users
+
+  resources :listings do
+    member do
+      get 'room'
+      get 'description'
+      get 'price'
+      get 'photos'
+      get 'amenity'
+      get 'location'
+      get 'preload'
+      get 'preview'
```

Comment:
Where did you get `preload` and `preview`?
This is not used in your current task.

Reply:
I forgot to remove this code, I was trying to make a preview and preload function to show the preview of the page before the user can publish it, but decided to scrap it since it wasn't part of the wireframe.